examples/runme.py

```diff
@@ -1,16 +1,18 @@
 from profiler import Profiler
 import pprint
 
-if __name__ == '__main__':
+def get_task_uids(num_pipelines):
+    
+    num_tasks = num_pipelines*7*1
+    task_uids = []
+    for t in range(num_tasks):
+        task_uids.append('radical.entk.task.%04d'%t)
 
-    p = Profiler()
+    return task_uids
 
-    print p.duration(objects= ['task.0000', 'task.0001'], states=['SCHEDULING', 'EXECUTED'])
-    print p.duration(
-        objects= ['wfprocessor.0000'], 
-        events=['starting dequeue-thread', 'terminating dequeue-thread'])
+if __name__ == '__main__':
 
+    task_uids = get_task_uids(16)
 
-    print p.duration(
-        objects= ['task.0000','wfprocessor.0000',], 
-        events=['starting dequeue-thread'], states=['EXECUTED'])
\ No newline at end of file
+    p = Profiler(src='./16_replicas/')
+    print p.duration(task_uids, states=['SCHEDULING','DONE'])
```

Comment:
do you actually have some constants defined with the state names anywhere, or do you always use the strings?